.github/workflows/ci_with_install.yml

```
@@ -31,31 +31,78 @@ jobs:
           apt-get --allow-releaseinfo-change update
           apt-get update -y
           apt-get upgrade -y
-          apt-get install -y libgl1-mesa-glx libgl1-mesa-dev libglu1-mesa-dev  freeglut3-dev libosmesa6 libosmesa6-dev  libgles2-mesa-dev
+          apt-get install -y libgl1-mesa-glx libgl1-mesa-dev libglu1-mesa-dev freeglut3-dev libosmesa6 libosmesa6-dev libgles2-mesa-dev libarchive-dev
           apt-get install -y libpangocairo-1.0-0
-          conda install -c conda-forge -y moab
-          conda install -c conda-forge -y gmsh
-          conda install -c conda-forge -y python-gmsh
-          conda install -c conda-forge -y "openmc=0.13.3=dagmc*nompi*"
+          conda install -c conda-forge mamba
+          mamba install -c conda-forge moab gmsh python-gmsh -y
+          mamba install -c "cadquery/label/dev" ocp=7.7.1
+          pip install git+https://github.com/CadQuery/cadquery.git@imprinting --no-deps 
```

```suggestion
          mamba install -c cadquery -c conda-forge cadquery=master
```
if imprinting PR gets merged we can change to this